app/src/containers/pages/Home/Home.tsx

```diff
@@ -43,6 +45,15 @@ const Home: React.FC<HomeProps> = (props: HomeProps) => {
             </Link>
           </Col>
         )}
+        {ENABLE_TEAMS && (
```

Comment:
Have we updated `app/src/containers/pages/Home/tests/Home.test.tsx` to  add a test case for when `ENABLE_TEAMS` is set to true?